report/src/problem_analysis/hci/prototyping.tex

```diff
@@ -0,0 +1,94 @@
+Figma contains a rich set of user interface components that can be used and modified when creating prototypes.
+When a user interacts with a hifi prototype, by, for instance, clicking a UI element, the designer can enable a transition to a different view, thus simulating system UI behavior.
+Based on the initial interviews described in Section \ref{sec:understanding_the_problem}, the prototypes must be suitable for use on a simple monitor, such as a tablet.
+
+During this project, both a lofi and hifi prototype has been developed.
+Each prototype shows a UI highlighting two situations for meeting rooms. 
+Each situation depicts different states of a meeting room (occupied or available) which allows us to explore how users want to interact with the system in these different situations. 
+
+
+\begin{figure}
+    \centering
+    \begin{subfigure}[b]{0.49\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{images/lofi_end.png}
+        \caption{Tablet view for when a meeting room is occupied.}
+        \label{fig:lofi_prototype:a}
+    \end{subfigure}
+    \begin{subfigure}[b]{0.49\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{images/lofi_schedule.png}
+        \caption{Tablet view for when a meeting room is available}
+        \label{fig:lofi_prototype:b}
+    \end{subfigure}
+    \caption{Two views of the low fidelity prototype, each enabling different actions based on the state of the meeting room.}
+    \label{fig:lofi_prototype}
+\end{figure}
+
+The development of a lofi prototype has been used as an internal exercise, independent from stakeholders, to explore initial design ideas based on existing systems.
+Figure \ref{fig:lofi_prototype} depicts two views from the prototype, each based on the current state of the meeting room.
+Depending on the state of the room, the background color as well as the actions available for the user changes.
+Figure \ref{fig:lofi_prototype:a} depicts a view shown when a meeting is occupied.
+Here, the main action available is to end the current meeting.
+Figure \ref{fig:lofi_prototype:b} depicts a view shown when a meeting room is available.
+From this view, the main action is to book the meeting room.
+
+Based on this prototype, multiple ideas for a hifi prototype were conceived.
+These ideas were filtered, refined, and turned into a single hifi prototype.
+Figure \ref{fig:hifi_prototype_interaction} depicts the view for the same situations shown in Figure \ref{fig:lofi_prototype:a} where a room is occupied.
+The hifi prototype enables user interaction with all elements that would be interactive in a final design.
+Each user interaction with an interactive element changes what is shown on the prototype. 
+This is shown in Figure \ref{fig:hifi_prototype_interaction}. 
+All views for the hifi prototype can be found in Appendix \ref{appendix:hifi_prototype}.
```

Comment:
Should we maybe just have shown just one image of the hifi version? So we don't just show the 'ugly' version in the primary text.

Reply:
The ugly version is a necessary step in many of the sources we use. It will seem weird that we jump from nothing to pretty design. The 'ugliness' is very okay in terms of HCI.

Reply:
My bad for being unclear. I did not mean that we should remove the first iteration, merely that I'd like to see the second iteration also in the main text.